### ISSUE TITLE
Code for project

### ISSUE THREAD

**Issue by manishvel8** [zenetio/DeepRL-Robotic#1]
Hi, can you please share the code for this project?
https://youtu.be/vxcCQmZHcXA?si=N_kX9_LgYRW_bz2n

**Comment by zenetio** [comment]
Yes, sure. I will post the link to my GitHub repository in the details.
Please take a look there.
________________________________
De: manishvel8 ***@***.***>
Enviado: terça-feira, 30 de janeiro de 2024 11:06
Para: zenetio/DeepRL-Robotic ***@***.***>
Cc: Subscribed ***@***.***>
Assunto: [zenetio/DeepRL-Robotic] Code for project (Issue #1)


Hi, can you please share the code for this project?
https://youtu.be/vxcCQmZHcXA?si=N_kX9_LgYRW_bz2n
